localstack/services/cloudformation/cloudformation_starter.py

```diff
@@ -575,6 +575,29 @@ def SNS_Topic_get_cfn_attribute(self, attribute_name):
     SNS_Topic_get_cfn_attribute_orig = sns_models.Topic.get_cfn_attribute
     sns_models.Topic.get_cfn_attribute = SNS_Topic_get_cfn_attribute
 
+    # Patch ES get_cfn_attribute(..) method
+    def ES_get_cfn_attribute(self, attribute_name):
+        if attribute_name in ['Arn', 'DomainArn']:
+            return aws_stack.es_domain_arn(self.params.get('DomainName'))
+        if attribute_name == 'DomainEndpoint':
+            if attribute_name == 'DomainEndpoint':
```

Comment:
nitpick: Looks like we have a duplicate check `if attribute_name == 'DomainEndpoint':` here

Reply:
I fixed it